src/test/java/com/microsoft/sqlserver/testframework/DBTable.java

```diff
@@ -82,6 +82,34 @@ public DBTable(boolean autoGenerateSchema, boolean unicode) {
         }
         this.totalColumns = columns.size();
     }
+    
+    /**
+     * Initializes {@link DBTable} with tableName, schema, and {@link DBColumns}
+     * 
+     * @param autoGenerateSchema
+     *            <code>true</code>: generates schema with all available
+     *            dataTypes in SqlType class
+     * @param unicode
+     *             <code>true</code>: sets unicode column names if autoGenerateSchema is also set to <code>true</code>
+     * @param alternateShcema
+     *             <code>true</code>: creates table with alternate schema
+     */
+    public DBTable(boolean autoGenerateSchema, boolean unicode, boolean alternateSchema) {
```

Comment:
We can reduced copy paste by changing `public DBTable(boolean autoGenerateSchema)` and `public DBTable(boolean autoGenerateSchema, boolean unicode)` 

Example: 
````
public DBTable(boolean autoGenerateSchema) {
 this(autoGenerateSchema, false,false);  
}
````